README.md

```diff
@@ -155,6 +155,7 @@ Check our [Roadmap][roadmap] for upcoming planned integrations.
 - [How does Kev differ from Kompose?](docs/tutorials/how-kev-differs-from-kompose.md)
 - [Getting started with Kev](docs/tutorials/getting-started-with-kev.md)
```

Comment:
I realise this is not part of this PR ... but maybe a rename ??
`Getting started with Kev` -> `Iterating with Kev`

Reply:
Getting started sounds more user friendly.